l10n_it_fatturapa_out/wizard/efattura.py

```diff
@@ -257,12 +257,12 @@ def _key(tax_id):
             out.update(out_computed)
             return out
 
-        def get_importo(line):
```

Comment:
Trovo più versatile passare l'intera line piuttosto che i due valori qui. Un sottomodulo potrebbe voler fare l'override dell'importo leggendo valori presi da line. Se passi solo price_unit e discount, altri moduli non ci possono fare molto.

Reply:
A me sembra più utile non duplicare il codice per il calcolo dello sconto.

Reply:
Beh prima c'era  funzione `get_importo()` che riceveva un parametro, `line`, adesso riceve due parametri, `price_unit` e `discount`, entrambi però presi da `line`.
È cambiata la signature della funzione, e rende la funzione meno versatile.


Forse sarebbe meglio passare un parametro aggiuntivo, opzionale, che permetta di selezionare quale sconto applicare, mantenendo la compatibilità con la base e altri (eventuali) moduli. Tipo:

```
def get_importo(line, discount_field="discount"):
  str_number = str(line[discount_field])
```
che permette di mantenere la compatibilità e comunque scegliere lo sconto da applicare, in modo del tutto generale. Un domani un altro modulo potrebbe voler fare dei conti sull'importo usando campi dalla `line`, o introdurre in altro campo sconto da usare come base tipo `discount4` (faccio per dire, non che abbia senso).

Reply:
Chiaro ma resto della mia idea; mi da l'impressione di un UCAS. Detto ciò, sentiti libero di modificare la pr.